src/containers/Devices.jsx

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux'
```

Comment:
Good idea to split the view part and the container elements I think here, but is it really better to maintain with many containers? What do the others think? @goldoraf? @gregorylegarec?